cluster/images/etcd-version-monitor/etcd-version-monitor.go

```diff
@@ -0,0 +1,275 @@
+/*
```

Comment:
Yes, I felt the same too. Changed it to download etcdctl (the only extra binary we need) as part of the makefile.